tests/components/application-connector/test/application-connectivity-validator/simple_test.go

```diff
@@ -0,0 +1,7 @@
+package application_connectivity_validator
```

Comment:
Please move TestGoodCert` and `TestBadCerts` into `simple_test.go `(or remove the file)